Assignment1/Assignment1_QNo1/SolAssignmentNo1_QNo1.rb

```diff
@@ -0,0 +1,16 @@
+# => 1. Find out how to convert the string "7.13" into the number 7.13
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def cvt_str_to_int
+	# => Repeateadly asking users to give correct input string
+	begin
+		puts "Enter another string"
+		str = gets.chomp
+		str_output = str.chomp.to_f
+	end until (str != nil && str_output != 0)
+	puts "The required floating conversion is #{str_output}"
+end 
```

Comment:
@HrisikeshSinghaChowdhury Please use two space instead of four space, configure your sublime.

Reply:
Assignment2 onwards i think it's resolved